### ISSUE TITLE
No content Found

### ISSUE THREAD

**Issue by Drucio** [magnuslsjoberg/The-Filipino-Channel.bundle#9]
Help!!? When I click a genre and try to find a show I get that error.

**Comment by nleoncio** [comment]
same thing happens to me. I was able to use it still last night bit for some reason today, it's not showing anything.

**Comment by magnuslsjoberg** [comment]
Well, I just got home and con confirm that TFC has changed a few things.
One thing that is changed is the login...

I will try to fix but I can't promise any update soon!

**Comment by ecalfonso** [comment]
@Drucio @nleoncio can you test this: https://github.com/ecalfonso/The-Filipino-Channel.bundle/commit/ed6183f184c492327ed0cf1699acf31d4423b59e

**Comment by nleoncio** [comment]
Hi. The Shows are now showing up on the page. I clicked Drama and it's
showing the recent shows. I tried playing and it's working fine as well.
Awesome, it's now working.

Niegel

Niegel Leoncio
Mobile: 949.415.4485
Email: niegel@gmail.com

**Comment by magnuslsjoberg** [comment]
@ecalfonso:

Thanks a lot for helping out!

I also found the div change and got the shows to show ;) but didn’t release any updates since my login fails.

I thought TFC had changed something in their login procedure as well and has been struggling to find what.

If others can login it must be something in my own setup that broke at the same time...

**Comment by magnuslsjoberg** [comment]
Arrgggghhh!! Spending many hours trying to find out what had changed in the login page when it turns out that login has worked all the time but Chrome's AutoFill replaced my plugin password when I changed the debug level and cache time in the settings.

@ecalfonso's correction is all that is needed! Thank you all for helping me in the right direction!

**Comment by Panday1980** [comment]
Please I need help with tfc, when I logged in and tried to play any tfc show it just spins and spins. What am I doing wrong.☹️

**Comment by nleoncio** [comment]
I tried to make sure my login works on tfc.tv. then update the plugin with
the new bundle. Then restart my plex server. Then reenter my tfc.tv
credentials. Make sure preview is not checked (i first tested it with that
checked which of course caused an issue). Then attempted to play a video.
It didnt work for the first time. I tried playing the video from prior day
then it worked. No further issue afterwards.
